components/stream-provider/package.json

```diff
@@ -52,6 +52,8 @@
       ]
     },
     "linux": {
+      "executableName": "browserd",
```

Comment:
there's some bugs in the debian electron-builder source, `executable` defaults to `name` and doesn't support `ext-macros` so we must switch it to the literal `browserd`. further, the debian builder uses an incompatible `artifactName` (again based on `name` not `productName` so we must manually define it, though it does support `ext-macros`).

Reply:
i see so it had to be changed several times